test/src/elit/boot_asset_fingerprint_test.clj

```diff
@@ -0,0 +1,40 @@
+(ns elit.boot-asset-fingerprint-test
+  (:require [boot.core :as boot]
+            [clojure.java.io :as io]
+            [clojure.string :as string]
+            [clojure.test :refer [deftest is testing]]
+            [elit.boot-asset-fingerprint :as sut]
+            [elit.fingerprint :as fingerprint]))
+
+(defn asset-refs
+  [text asset-host]
+  (->> (re-seq (re-pattern (str asset-host ".+\\.(css|js)"))
+               text)
+       (map first)))
+
+(defn remove-asset-host
+  [asset-ref asset-host]
+  (string/replace-first asset-ref (re-pattern asset-host) ""))
+
+#_
+;; doesn't work when triggered by a boot task
```

Comment:
Can this test be fixed? If not, is it worth just deleting it?

Reply:
I hope so. Need to figure out how to e2e test boot tasks.